go-controller/pkg/ovn/endpoints.go

```diff
@@ -148,29 +147,11 @@ func (ovn *Controller) handleNodePortLB(node *kapi.Node) error {
 			continue
 		}
 		for _, ep := range endpoints {
-			svc, err := ovn.watchFactory.GetService(ep.Namespace, ep.Name)
-			if err != nil {
-				continue
-			}
-			if !util.ServiceTypeHasNodePort(svc) {
-				continue
-			}
-			protoPortMap := ovn.getLbEndpoints(ep)
-			for _, svcPort := range svc.Spec.Ports {
-				lbEps, isFound := protoPortMap[svcPort.Protocol][svcPort.Name]
-				if !isFound {
-					continue
-				}
-				k8sNSLb, _ := ovn.getGatewayLoadBalancer(gatewayRouter, svcPort.Protocol)
-				if k8sNSLb == "" {
-					return fmt.Errorf("%s load balancer for node %q does not yet exist", svcPort.Protocol, node.Name)
-				}
-				err = ovn.createLoadBalancerVIPs(k8sNSLb, physicalIPs, svcPort.NodePort, lbEps.IPs, lbEps.Port)
-				if err != nil {
-					klog.Errorf("Failed to create VIP in load balancer %s - %v", k8sNSLb, err)
-					continue
-				}
+			if err := ovn.AddEndpoints(ep, false); err != nil {
```

Comment:
yes, a lot of the legacy controller names dont make sense. I really dont need support in the old controller, since we switched to new controllers. I just started down this path before the new controller was ready. I think its OK to keep the code in the legacy controllers just in case for some reason we have to revert to the old controller.

That being said I dont really want to change the function name as it is well known to a bunch of us that have worked on the legacy controllers. I think updating this may just add more confusion down the road if we have go and debug a legacy issue.